js/interface.js

```diff
@@ -1,78 +1,176 @@
-var $folderContents = $('#folder-contents');
+// VAR SETUP //
+var $folderContents = $('.file-table-body');
+var $folderList;
+var $organizationAppList = $('.dropdown-menu-holder ul');
 var templates = {
   file: template('file'),
-  folder: template('folder')
+  folder: template('folder'),
+  folderItem: template('folder-item'),
+  organizations: template('organizations'),
+  apps: template('apps')
 };
+
 var currentFolderId;
 var currentFolders;
 var currentFiles;
+var counter;
+
+// CORE FUNCTIONS //
+// Get organizations and apps list for left side menu
+function getOrganizationsList() {
+  counter = 0;
+  Fliplet.Organizations.get().then(function (organizations) {
+    organizations.forEach(addOrganizations);
+    $folderList = $('.folder-list');
+  });
+}
+
+function getAppsList() {
+  Fliplet.Apps.get().then(function (apps) {
+    apps.forEach(addApps);
+    $folderList = $('.folder-list');
+  });
+}
+
+// Get folders and files depending on ID (Org, App, Folder) to add to the content area
+function getFolderContents(el) {
+  if (el.attr('data-type') == "app") {
+    var currentAppId = el.attr('data-app-id');
```

Comment:
every time the `getFolderContents()` runs I need those variables to be empty, otherwise they will be added as options to the `.get()` API. I just want to pass one option, that is why I that way only pass that option of the condition.